main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1997,8 +1997,8 @@ public static void startActivity(final Context context, final String geocode, fi
         context.startActivity(cachesIntent);
     }
 
-    private ActionMode mActionMode = null;
-    private boolean mSelectionModeActive = false;
+    private ActionMode actionMode = null;
+    private boolean selectionModeActive = false;
```

Comment:
Yeah, sorry about that.  I just got a little overzealous with removing my 'm' prefixes and strayed into someone else's work ;)

I'll undo it again.